sdk/typescript/src/zklogin/address.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { blake2b } from '@noble/hashes/blake2b';
+import { bytesToHex, utf8ToBytes } from '@noble/hashes/utils';
+
+import { SIGNATURE_SCHEME_TO_FLAG } from '../cryptography/signature-scheme.js';
+import { normalizeSuiAddress, SUI_ADDRESS_LENGTH } from '../utils/index.js';
+import { toBufferBE } from './utils.js';
+
+export function computeZkLoginAddressFromSeed(addressSeed: bigint, iss: string) {
+	const addressSeedBytesBigEndian = toBufferBE(addressSeed, 32);
+	const addressParamBytes = utf8ToBytes(iss);
```

Comment:
we are already using TextEncoder other places, so we could probably avoid this dependency here